open_schools_platform/organization_management/organizations/tests/views/test_invite_employee.py

```diff
@@ -0,0 +1,28 @@
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework.test import APIClient
+
+from open_schools_platform.organization_management.employees.selectors import get_employee_profile, get_employee
+from open_schools_platform.query_management.queries.models import Query
+from open_schools_platform.user_management.users.tests.utils import create_logged_in_user
+
+
+class InviteEmployeeTests(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.invite_employee_url = \
+            lambda pk: reverse("api:organization-management:organizations:invite-employee", args=[pk])
+
+    def test_invite_employee_query_successfully_formed(self):
+        user = create_logged_in_user(instance=self)
+        data = {
+            "phone": user.phone,
```

Comment:
what phone do we use there?

Reply:
if it is testing phone number so it is ok

Reply:
Yes, in create_logged_in_user function we create user with phone number that is listed in firebase as a test one